app/controllers/classified_listings_controller.rb

```diff
@@ -9,69 +9,63 @@ def index
     @displayed_classified_listing = ClassifiedListing.find_by!(slug: params[:slug]) if params[:slug]
     mod_page if params[:view] == "moderate"
     @classified_listings = if params[:category].blank?
-                             ClassifiedListing.where(published: true).order("bumped_at DESC").limit(12)
+                             ClassifiedListing.where(published: true).
+                               order("bumped_at DESC").
+                               includes(:user, :organization, :taggings).
+                               limit(12)
                            else
-                             []
+                             ClassifiedListing.none
                            end
     set_surrogate_key_header "classified-listings-#{params[:category]}"
   end
 
   def new
     @classified_listing = ClassifiedListing.new
     @organizations = current_user.organizations
-    @credits = current_user.credits.where(spent: false)
+    @credits = current_user.credits.unspent
   end
 
   def edit
     authorize @classified_listing
     @organizations = current_user.organizations
-    @credits = current_user.credits.where(spent: false)
+    @credits = current_user.credits.spent
```

Comment:
Is it intended that the `spent` credits are queued instead of `unspent`?

Reply:
No, definitely an error, thanks!